docs/security.md

```diff
@@ -258,3 +258,55 @@ If you want to use the command `curl`, you can follow the commands:
 openssl x509 -inform der -in $JAVA_HOME/localhost.crt -out certificate.pem
 curl -v -X GET --cacert ./certificate.pem -H "Accept: application/vnd.gravitino.v1+json" -H "Content-Type: application/json" https://localhost:8433/api/version
 ```
+
+## Custom filters
+
+Gravitino supports custom filters to implement the user specified logic to satisfy different safety needs.
+
+### Gravitino server's configuration
+
+| Configuration item                          | Description                                                      | Default value | Required | Since version |
+|---------------------------------------------|------------------------------------------------------------------|---------------|----------|---------------|
+| `gravitino.server.webserver.customFilters`  | Comma separated list of filter class names to apply to the APIs. | ``            | No       | 0.4.0         |
+
+The filter should be a standard javax servlet Filter.
+Filter parameters can also be specified in the configuration, by setting config entries of the form `gravitino.server.webserver.<class name of filter>.param.<param name>=<value>`
+
+For example:
+```text
+gravitino.server.webserver.customFilters=com.test.filter1
+gravitino.server.webserver.com.test.filter1.param.name1=foo
+gravitino.server.webserver.com.test.filter1.param.name2=ba
+```
+
+### Iceberg REST service's configuration
+
+| Configuration item                                  | Description                                                        | Default value | Required | Since version |
+|-----------------------------------------------------|--------------------------------------------------------------------|---------------|----------|---------------|
+| `gravitino.auxService.iceberg-rest.customFilters`   | Comma separated list of filter class names to apply to the APIs.   | ``            | No       | 0.4.0         |
+
+The filter should be a standard javax servlet Filter.
+Filter parameters can also be specified in the configuration, by setting config entries of the form `gravitino.auxService.iceberg-rest.<class name of filter>.param.<param name>=<value>`
+
+For example:
+```text
+gravitino.auxService.iceberg-rest.customFilters=com.test.filter1
+gravitino.auxService.iceberg-rest.com.test.filter1.param.name1=foo
+gravitino.auxService.iceberg-rest.com.test.filter1.param.name2=ba
+```
+
+### Example
+
+If you want to use a cross-origin filter for the Gravitino server, you can follow the steps.
+
+```shell
+wget https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-servlets/9.4.51.v20230217/jetty-servlets-9.4.51.v20230217.jar 
+cp jetty-servlets-9.4.51.v20230217.jar <gravitino home>/libs
```

Comment:
Can you please change to `<GRAVITINO_HOME>/libs`

Reply:
OK. will fix.

Reply:
Fixed.